validator/src/main/resources/validations/java/ecs/log-validation.yml

```
@@ -0,0 +1,4 @@
+-
+  validationType: "cw-log"
+  callingType: "none"
+  expectedLogStructureTemplate: "JAVA_ECS_HC_CALL_LOG"
```

If we're only doing one API call, then we can merge the validation files into one. Definitely not needed, having them separate is also valid so that they can be easier to debug, but just letting you know it's an option:
```suggestion
-
  validationType: "cw-log"
  callingType: "none"
  expectedLogStructureTemplate: "JAVA_ECS_HC_CALL_LOG"
-
  validationType: "cw-metric"
  callingType: "none"
  expectedMetricTemplate: "JAVA_ECS_HC_CALL_METRIC"
-
  validationType: "trace"
  callingType: "none"
  expectedTraceTemplate: "JAVA_ECS_HC_CALL_TRACE"
```

---

Got it. Thanks for the info. Let's continue to separate them follow the other platforms.